src/main/java/uk/gov/hmcts/reform/unspec/handler/callback/ConfirmServiceCallbackHandler.java

```diff
@@ -30,18 +35,19 @@
 import static uk.gov.hmcts.reform.unspec.helpers.DateFormatHelper.formatLocalDateTime;
 
 @Service
-@RequiredArgsConstructor
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
```

Comment:
`(onConstructor = @__(@Autowired))` is not required.
```suggestion
@RequiredArgsConstructor
```